Source/engine.h

```diff
@@ -197,6 +197,18 @@ struct Point {
 
 		return (approx + 512) / 1024;
 	}
+
+	Point Absolute()

```

Comment:
From what I've gathered, usually libraries override the `abs` function to do this. I've seen multiple libraries from STD library do it, for example "complex numbers" have an `abs` method.

I'd recommend following the same approach and using the same name here.

EDIT: Of course, it would also change into a static method instead of an instance one.